### DISCUSSION TITLE
Cover Page Title

### DISCUSSION THREAD

**Discussion by ellenarchie** [thegetty/quire#236] [Get Help] [answered]
Is there a way to make the title invisible on the title page and only show up on the Table of Contents and Side Bar menu? I'd like to upload a cover image with a stylized title included on the image itself.

**Comment by Erin-Cecele** [comment, marked as answer]
Hi @ellenarchie! Happy to help. In the `static` folder of your project you'll find a `css` folder. In the `css` folder you'll see a file called `custom.css`. Scroll to the bottom and add the following CSS to hide the title, subtitle, and contributors. While those fields will be hidden visually, they will still be accessible to anyone using a screen reader:
 
```
quire-cover__hero-body {
  clip: rect(0 0 0 0);
  clip-path: inset(50%);
  height: 1px;
  overflow: hidden;
  position: absolute;
  white-space: nowrap;
  width: 1px;
}
 ```

Here is a little more information on using custom CSS in Quire: https://quire.getty.edu/documentation/styles-customization/#add-custom-styles. 

Let us know if you have questions or need additional assistance.

**Reply by Erin-Cecele** [reply]
Hi @ellenarchie. Did this help resolve your issue?

**Reply by ellenarchie** [reply]
Thank you so much for your help. I initially still had trouble, but I have been able to show your reply to someone on my end and we added a period at the beginning, and now it works!  The first line now reads like this: 
.quire-cover__hero-body {

Thank you so much

**Reply by Erin-Cecele** [reply]
That's wonderful, @ellenarchie. Thanks for confirming. Let us know if you need anything additional.

**Reply by mmpadilla15** [reply]
Hi @Erin-Cecele - I just tried your fix and added the period at the beginning of the first line and it's not working for me. But, in your initial answer, you advise to scroll to the bottom of the custom.css doc to add the code. When I opened my custom.css doc, there wasn't anything in it so there wasn't anything to scroll to.

Any advice?

**Reply by Erin-Cecele** [reply]
Hi @mmpadilla15. That's fine! That just means you haven't added any custom CSS yet. Some other users make a lot of customizations. Just cut and paste the text into your custom.css file and see if that helps.

**Reply by mmpadilla15** [reply]
@Erin-Cecele Nope - I've tried it with and without the period at the beginning of the first line and no luck yet.

**Comment by Erin-Cecele** [comment]
Hmm @mmpadilla15 it just worked for me. Can you send me a screenshot of your custom.css file? You definitely want to have the period there. It should look like this:

```
.quire-cover__hero-body {
  clip: rect(0 0 0 0);
  clip-path: inset(50%);
  height: 1px;
  overflow: hidden;
  position: absolute;
  white-space: nowrap;
  width: 1px;
}
```

**Reply by mmpadilla15** [reply]
@Erin-Cecele Sure! Here you go:
![Screen Shot 2022-08-25 at 2 42 36 PM](https://user-images.githubusercontent.com/74433007/186754059-9d971b53-4533-4c7c-9486-90b51df578b2.png)

**Reply by Erin-Cecele** [reply]
@mmpadilla15 Looks good! Have you tried closing Quire down with the command `quire stop` and previewing it again? Sometimes it takes a hard refresh to force a change to become visible. You can also try previewing in a different browser.

**Reply by mmpadilla15** [reply]
@Erin-Cecele OK, I just did `quire stop`, then `quire install`, then `quire preview` and the title box is still there.

**Reply by Erin-Cecele** [reply]
@mmpadilla15 It might be best to hop on a screen-sharing call. I will email you to set up a time!
